--- phpMyAdmin-2.5.5/config.inc.php~	2003-12-29 01:02:27.000000000 +0100
+++ phpMyAdmin-2.5.5/config.inc.php	2003-12-29 01:02:55.000000000 +0100
@@ -79,7 +79,7 @@
                                                     // The controluser is also
                                                     // used for all relational
                                                     // features (pmadb)
-$cfg['Servers'][$i]['auth_type']     = 'config';    // Authentication method (config, http or cookie based)?
+$cfg['Servers'][$i]['auth_type']     = 'cookie';    // Authentication method (config, http or cookie based)?
 $cfg['Servers'][$i]['user']          = 'root';      // MySQL user
 $cfg['Servers'][$i]['password']      = '';          // MySQL password (only needed
                                                     // with 'config' auth_type)
