diff -uNr phpMyAdmin-2.5.6-rc1/config.inc.php phpMyAdmin-2.5.6-rc1.new/config.inc.php
--- phpMyAdmin-2.5.6-rc1/config.inc.php	Fri Jan  2 20:31:58 2004
+++ phpMyAdmin-2.5.6-rc1.new/config.inc.php	Sat Feb  7 19:17:43 2004
@@ -79,8 +79,8 @@
                                                     // The controluser is also
                                                     // used for all relational
                                                     // features (pmadb)
-$cfg['Servers'][$i]['auth_type']     = 'config';    // Authentication method (config, http or cookie based)?
-$cfg['Servers'][$i]['user']          = 'root';      // MySQL user
+$cfg['Servers'][$i]['auth_type']     = 'cookie';    // Authentication method (config, http or cookie based)?
+$cfg['Servers'][$i]['user']          = 'mysql';      // MySQL user
 $cfg['Servers'][$i]['password']      = '';          // MySQL password (only needed
                                                     // with 'config' auth_type)
 $cfg['Servers'][$i]['only_db']       = '';          // If set to a db-name, only
