*** libraries/config.default.php.orig	Tue Aug 19 00:56:44 2008
--- libraries/config.default.php	Tue Aug 19 00:56:08 2008
***************
*** 80,85 ****
--- 80,92 ----
  $i = 1;
  
  /**
+  * Display name
+  *
+  * @global string $cfg['Servers'][$i]['display']
+  */
+ $cfg['Servers'][$i]['display'] = 'localhost';
+ 
+ /**
   * MySQL hostname or IP address
   *
   * @global string $cfg['Servers'][$i]['host']
