Add 'display' configuration option for server selection label.

--- phpMyAdmin-5.2.3-all-languages/libraries/config.default.php.orig	2025-10-07 22:40:00.000000000 +0200
+++ phpMyAdmin-5.2.3-all-languages/libraries/config.default.php	2025-10-07 22:40:00.000000000 +0200
@@ -125,6 +125,13 @@
 $i = 1;

 /**
+ * Display name
+ *
+ * @global string $cfg['Servers'][$i]['display']
+ */
+$cfg['Servers'][$i]['display'] = 'localhost';
+
+/**
  * MySQL hostname or IP address
  *
  * @global string $cfg['Servers'][$i]['host']
