--- phpMyAdmin-2.11.0-all-languages.orig/libraries/config.default.php	2007-08-22 20:04:58.000000000 +0200
+++ phpMyAdmin-2.11.0-all-languages/libraries/config.default.php	2007-08-22 20:05:57.000000000 +0200
@@ -141,14 +141,14 @@
  *
  * @global string $cfg['Servers'][$i]['auth_type']
  */
-$cfg['Servers'][$i]['auth_type'] = 'config';
+$cfg['Servers'][$i]['auth_type'] = 'cookie';
 
 /**
  * MySQL user
  *
  * @global string $cfg['Servers'][$i]['user']
  */
-$cfg['Servers'][$i]['user'] = 'root';
+$cfg['Servers'][$i]['user'] = 'mysql';
 
 /**
  * MySQL password (only needed with 'config' auth_type)
